Add product to Getting started and update Admins instructions (#1357)
RoadieHQ/marketing-site: content/docs/integrations/jenkins/index.md

## Prerequisites

> 1. You must be an admin in Roadie. By default, all users are admins. Learn how to designate certain users as admins [here](/docs/getting-started/assigning-admins/).

Reviewer: ```suggestion
1. You must be an admin in Roadie. By default, the first Roadie user is an admin. Learn how to designate certain users as admins [here](/docs/getting-started/assigning-admins/).
```
